flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -83,6 +83,17 @@
 				code("org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler"))
 				.build());
 
+	/**
+	 * The scheduler maximum-allocation-vcores exposed by YARN. This is a global property of the Yarn scheduler."
+	 */
+	public static final ConfigOption<Integer> SCHEDULER_MAX_VCORE =
```

Comment:
Why do we need this config options?

Reply:
Thanks for catching this. I was experimenting something else(overwrite). Removed.